core/templates/dev/head/css/oppia.css

```diff
@@ -1358,6 +1358,20 @@ pre.oppia-pre-wrapped-text {
   text-align: left;
 }
 
+.protractor-test-state-content-display > p {
+  margin-top: 18px;
+  margin-bottom: 18px;
+  line-height: 28px;
```

Comment:
nit: for each rule, alphabetize css attrs -- so, line-height before margin-bottom before margin-top. Ditto below. (Makes it easy to find attrs if there are many of them.)